sbin/changelog.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
- [ ] Would be great if this and next scripts were a separate npm package as I'm using them in `react-hooks-shareable` repo

Reply:
I wonder if we should add BREAKING CHANGES information as well when generating the changelog?

Reply:
I would just copy the scripts, they are tiny. If we do something (much) more fancy we might consider having a separate project.